test/e2e/dra/test-driver/deploy/example/admin-access-policy.yaml

```diff
@@ -25,7 +25,7 @@ spec:
   validations:
     - expression: '! object.spec.devices.requests.exists(e, has(e.adminAccess) && e.adminAccess)'
       reason: Forbidden
-      messageExpression: '"admin access to devices not enabled"' # in namespace " + object.metadata.namespace' - need to use __namespace__, but somehow that also doesn't work.
```

Comment:
I explained the rationale for this change here: https://github.com/kubernetes/kubernetes/pull/129514#discussion_r1907511287